test/e2e/graceful_shutdown_test.go

```diff
@@ -27,8 +34,6 @@ func TestNodeCAGracefulShutdown(t *testing.T) {
 		t.Fatalf("unable to list pods: %v", err)
 	}
 
```

Comment:
I was looking at the error being reported by CI on this test and it seems like we could not find any pod to attach to(that would be very strange). Would you mind incorporating this slight change to make things clear? 

```suggestion
	if len(pods.Items) == 0 {
		t.Fatalf("no node-ca pods found")
	}
```